app/routes/webhooks.customers.data_request.tsx

```diff
@@ -0,0 +1,18 @@
+import type { ActionFunctionArgs } from "@remix-run/node";
+import { authenticate } from "~/shopify.server";
+
+export const action = async ({ request }: ActionFunctionArgs) => {
+  const { session } = await authenticate.webhook(request);
+
+  // Webhook requests can trigger after an app is uninstalled
+  // If the app is already uninstalled, the session may be undefined.
+  if (!session) {
```

Comment:
Someone from the webhooks team should confirm this. But I am wondering if we should be able to process all privacy webhook types without a session/installed shop.